### ISSUE TITLE
How to consume KRAGEN with LLM?

### ISSUE THREAD

**Issue by atljoseph** [EpistasisLab/KRAGEN#84]
Is there an example of KRAGEN being used by an LLM? Looks like a cool project, but having trouble seeing where “the rubber meets the road”. Thank you

**Comment by miguelehernandez** [comment]
Dear user, 
Yes, there is an example to run some queries described at the end of the README.md in this repo. Please check it out and let me know if you still have questions. Here is the text: 

"If you used the sample dataset, try asking "what genes are associated with caffeine?" and analyze the thought flow!"

Very straight forward... Let us know!
Miguel

**Comment by huycke** [comment]
ChatGPT is a large language model. Are you talk about a local model, one being run on your computer?

**Comment by atljoseph** [comment]
Yes. Local. Ollama. But also, is this something to be used in another tool,
or encapsulated to itself?

**Comment by chozillla** [comment]
I also am in need of Ollama support or something not reliant on an API
